code/modules/mob/living/carbon/human/species/outsider/undead.dm

```diff
@@ -237,6 +235,17 @@
 	H.accent = ACCENT_BLUESPACE
 	return ..()
 
+//Zombies do not have oxygen, so we have to handle the sprint this way
+/datum/species/zombie/handle_sprint_cost(var/mob/living/carbon/human/H, var/cost, var/pre_move)
+	. = ..()
+
+	if(exhaust_threshold && (H.stamina) < (exhaust_threshold * 0.8))
+		H.m_intent = M_WALK
+		H.hud_used.move_intent.update_move_icon(H)
+		to_chat(H, SPAN_DANGER("You're too exhausted to run anymore!"))
```

Comment:
```suggestion
		to_chat(H, SPAN_WARNING("You're too exhausted to run anymore!"))
```
This is a failure state, not a damage message. Just warn the player.

Reply:
This is a copied message from the function that it calls up, which uses span danger